source/manual/republishing-content.html.md.erb

```diff
@@ -58,6 +58,7 @@ To republish all instances of the following document types, run the following ra
 - TopicalEvent
 - TopicalEventAboutPage
 - WorldLocation
```

Comment:
Does world location need removing if it's not a proper content type any more?

Reply:
It needs to stay as the International Delegation pages can only be republished by republishing the `WorldLocation` model.